src/userProfile/Profile.jsx

```diff
@@ -123,17 +116,53 @@ export const Profile = () => {
               </PrimaryText>
             </TextStyles>
             <AvatarsImagesStyles>
-              <SelectedAvatar avatar={dog} />
+              <SelectedAvatar name='selectedAvatar' avatar={currentAvatar} />
               <AvatarsOptionsStyles>
-                <Avatar avatar={dj} />
-                <Avatar avatar={dog} />
-                <Avatar avatar={watch} />
-                <Avatar avatar={train} />
-                <Avatar avatar={whatever} />
-                <Avatar avatar={priestess} />
-                <Avatar avatar={ghost} />
-                <Avatar avatar={dude} />
-                <Avatar avatar={pig} />
+                <Avatar
```

Comment:
If we would put the img names in an array, could we use these array not for the avatar and the name? and shorten the code if we map through it?

Reply:
I thought about it, but it could make the code more obscure. We are going to have always nine avatars, so that cannot grow, and array suggest we can add and add values. I know the css looks long, but when there is too many components that is normal.

Reply:
With Typescript we could define another kind of structure to secure what can array we are going to have, but JS it too dynamic and we cannot do that.

Reply:
it would be something like :  
```const avatar = ['dj' ,'dog','watch', 'train', 'whatever', 'priestess','ghost','dude','pig' ]
const getAvatar = avatar.map((item)=> <Avatar name={`${item}`}avatar={`${user.baseAvatarUrl}/{${item}.png`}  onClick={onSelectAvatar}/> ```

this is more obscure?

Reply:
It is obscure because it is not clear that it can be just 9, it is also not clear which are the values of the array. You are thinking we are alone, but usually you have to think about a team. If someone sees that, doesn't know anything about the array.

Reply:
Even ourselves in 2 months or even 2 weeks won't remember why we did that. What are you saving with that? CSS? Remember those images are not changing, that is important here.